doc/whats_new.rst

```diff
@@ -110,6 +110,8 @@ Bug
 
 - Fix event sample number increase when combining many Epochs objects with :func:`mne.concatenate_epochs` with  by `Jasper van den Bosch`_
 
+- Fix error in mne coreg when saving with scaled MRI if fiducials haven't been saved by `Ezequiel Mikulan`_
```

Comment:
Almost the right pattern matching here :)

You need to add an entry at the bottom so that there is a URL